components/kyma-environment-broker/internal/process/input/input.go

```diff
@@ -428,7 +428,9 @@ func (r *RuntimeInput) applyProvisioningParametersForProvisionRuntime() error {
 	updateInt(&r.provisionRuntimeInput.ClusterConfig.GardenerConfig.AutoScalerMax, params.AutoScalerMax)
 	updateInt(r.provisionRuntimeInput.ClusterConfig.GardenerConfig.VolumeSizeGb, params.VolumeSizeGb)
 	updateString(&r.provisionRuntimeInput.ClusterConfig.GardenerConfig.Name, r.shootName)
-	updateString(&r.provisionRuntimeInput.ClusterConfig.GardenerConfig.Region, params.Region)
+	if params.Region != nil && *params.Region != "" {
```

Comment:
I'm thinking about having this "if" statement inside the updateString function, but not sure if it breaks something

Reply:
yeah, I agree that for some arguments there could be a difference between providing empty value or not providing value at all, probably safer to keep it separate.